test/screenplay/permanent_config_test.exs

```diff
@@ -373,45 +376,56 @@ defmodule Screenplay.PermanentConfigTest do
 
       File.write(pending_screens_path, config)
 
-      on_exit(fn ->
-        fresh_config = [%{id: "place-test", name: "Test Place", screens: [], routes: ["Green-B"]}]
+      [
+        %Place{
+          id: "place-test",
+          name: "Test Place",
+          routes: ["Green-B"],
+          screens: [],
+          description: nil
+        }
+      ]
+      |> Enum.map(&{&1.id, &1})
+      |> Cache.put_all()
 
-        File.write(get_fixture_path("places_and_screens.json"), Jason.encode!(fresh_config))
+      on_exit(fn ->
+        Cache.delete_all()
       end)
     end
 
     test "publishes pending screens and adds screen to places_and_screens" do
```

Comment:
The "adds screen to places_and_screens" and "does not add to places_and_screens" parts of these tests appear to no longer be tested. I think that's fine as that's no longer part of this module's responsibility, but it does mean the second test might be redundant now, and the descriptions should be updated.